### ISSUE TITLE
[regression] bannerBmp and dialogBmp became required attributes of Msi task in 4.8.10

### ISSUE THREAD

**Issue by slonopotamus** [i-net-software/SetupBuilder#105]
```
Some problems were found with the configuration of task ':msi' (type 'Msi').
> No value has been specified for property 'bannerBmp'.
> No value has been specified for property 'dialogBmp'.
```

See https://github.com/slonopotamus/stevedore/pull/2/checks?check_run_id=3507915997#step:4:43

These properties were optional in 4.8.7 and I believe they should be optional again.

Googling tells me this can be achieved by adding `@Optional` annotation to `Msi.getBannerBmp` and `Msi.getDialogBmp` methods.

Broken by 413900db4e9fae4f9e92fbea7e1b6355705aa929

**Comment by jvmlet** [comment]
Which gradle version are you using?

**Comment by Horcrux7** [comment]
Should be fixed with #104

**Comment by slonopotamus** [comment]
I'm on Gradle 6.9
